### PR TITLE
psionic messages

### PR DESCRIPTION
**Pull request by Trilbyspaceclone**
Ghosts now can toggle off psionic messages and the churchies sending message
Psionic messages is more vage i.e not only voice

### CODE REVIEW

**Comment by Lamasmaster** [comment]
Finally a good PR

File: code/modules/client/preference_setup/global/preferences.dm [hunk 1]
```diff
@@ -113,6 +114,11 @@ var/list/_client_preferences_by_type
 	key = "CHAT_GHOSTEARS"
 	options = list(GLOB.PREF_ALL_SPEECH, GLOB.PREF_NEARBY)
 
+/datum/client_preference/ghost_ears_plus
+	description ="Ghost Psionics/Cruciform"
+	key = "CHAT_GHOST_EARS_PLUS"
+	options = list(GLOB.PREF_HIDDEN_SPEECH, GLOB.PREF_NEARBY)
```
Comment (Trilbyspaceclone) [inline code/modules/client/preference_setup/global/preferences.dm:120]:
```suggestion
	options = list(GLOB.PREF_YES, GLOB.PREF_NO)
```

File: code/modules/psionics/psion_powers/psion_starter_powers.dm [hunk 2]
```diff
@@ -23,9 +23,10 @@
 
 	if(pay_power_cost(psi_point_cost) && check_possibility(TRUE, target))
 		var/say = sanitize(input("What do you wish to say"))
-		target.show_message("\blue <b><font size='3px'> You hear [usr.real_name]'s voice: [say] </font></b>")
+		target.show_message("\blue <b><font size='3px'> [usr.real_name]'s thoughts are projected into your mind: [say] </font></b>")
 		usr.show_message("\blue You project your mind into [target.real_name]: [say]")
 		log_say("[key_name(usr)] sent a telepathic message to [key_name(target)]: [say]")
 		for(var/mob/observer/ghost/G in world)
-			G.show_message("<i>Telepathic message from <b>[owner]</b> to <b>[target]</b>: [say]</i>")
+			if(G.get_preference_value(/datum/client_preference/ghost_ears) == GLOB.PREF_HIDDEN_SPEECH)
```
Comment (Trilbyspaceclone) [inline code/modules/psionics/psion_powers/psion_starter_powers.dm:30]:
```suggestion
			if(G.get_preference_value(/datum/client_preference/ghost_ears) == GLOB.GLOB.PREF_YES)
```

File: code/modules/client/preference_setup/global/preferences.dm [hunk 3]
```diff
@@ -1,6 +1,7 @@
 GLOBAL_VAR_CONST(PREF_YES, "Yes")
 GLOBAL_VAR_CONST(PREF_NO, "No")
 GLOBAL_VAR_CONST(PREF_ALL_SPEECH, "All Speech")
+GLOBAL_VAR_CONST(PREF_HIDDEN_SPEECH, "Hidden Speech")
```
Comment (Trilbyspaceclone) [inline code/modules/client/preference_setup/global/preferences.dm:4]:
```suggestion
```

File: code/modules/core_implant/cruciform/rituals/base.dm [hunk 4]
```diff
@@ -158,6 +158,10 @@
 	log_and_message_admins("[user.real_name] sent a message to [H] with text \"[text]\"")
 	playsound(user.loc, 'sound/machines/signal.ogg', 50, 1)
 	playsound(H, 'sound/machines/signal.ogg', 50, 1)
+	for(var/mob/observer/ghost/G in world)
+		if(G.get_preference_value(/datum/client_preference/ghost_ears) == GLOB.PREF_HIDDEN_SPEECH)
```
Comment (Trilbyspaceclone) [inline code/modules/core_implant/cruciform/rituals/base.dm:162]:
```suggestion
		if(G.get_preference_value(/datum/client_preference/ghost_ears) == GLOB.GLOB.PREF_YES)
```

**Review by Trilbyspaceclone — COMMENTED** [review]
Fixes needed

**Pull request merged**